src/main/java/de/tum/in/www1/bamboo/server/ResultsContainer.java

```diff
@@ -59,13 +60,7 @@ public long getInitTimestamp() {
 
     @Override
     public String toString() {
-        return "ResultsContainer{" +
```

Comment:
I actually preferred the old formatting here. Can we disable the formatter for this part?

Reply:
I have to look into the options file.

Reply:
Found the option